Jenkinsfile

```diff
@@ -110,7 +110,7 @@ pipeline {
   }
   post {
     always {
-      sh 'make clean'
+      sh 'make ci-down'
```

Comment:
I think we also want `make ci-clean` here as well, no?

Reply:
nah, the following `deleteDir()` whacks all the data, the clean was there to run from inside the container to remove the root owned files which `deleteDir()` could not because it is run as the jenins user.  

We don't have root owned files anymore because of the user changes to run as vets-api user.

Reply:
Which changes are you referring to specifically? 

I too had once believed we were running everything as the `vets-api`, but I eventually gave up and accepted that, `root` was _somehow_  coming to own various files.

Reply:
The old Dockerfile did a user creation in the container (eg: the `useradd`), but never set the USER directive in the Dockerfile to make docker use it, so it kept running things as root.  I added USER directives in the stages where they are needed to make it run as vets-api user.

That is also the reason for the `userid` ARG that gets passed in via the `VETS_API_USER_ID` env var set in the Makefile.  That will override the default userid of the vets-api user in the container and make it match the person building and running the container .  That way that the files written to the mounted volume have the same UID inside and outside the container and can be cleaned up easily.